### DISCUSSION TITLE
Bibliothèque famille - partager l’application

### DISCUSSION THREAD

**Discussion by pbillerot** [mtotschnig/MyExpenses#1126] [Français]
J’ai acheté l’application myexpenses pour synchroniser les données dans le drive de Google. J’ai installée l’application sur une tablette et mon smartphone. La synchronisation fonctionne très bien. Sur ma tablette, je voudrai partager l’application avec mon épouse avec chacun son profil et compte google différent. J’ai créé une bibliothèque famille sur le play store. L’application myexpenses à priori n’est pas candidate au partage. Comment puis-je faire pour partager la même licence ?

**Comment by mtotschnig** [comment]
Malheureusement, c'est Google qui ne permet pas d'inclure les achats effectués à l'intérieur d'une application dans la bibliothèque familiale:
https://support.google.com/googleplay/answer/7007852?hl=fr

> Vous ne pouvez pas partager les achats via une application
